packages/assets-controllers/package.json

```diff
@@ -50,6 +50,7 @@
     "async-mutex": "^0.2.6",
     "cockatiel": "3.1.1",
     "ethereumjs-util": "^7.0.10",
+    "events": "^3.3.0",
```

Comment:
@Gudahtt Does it make sense to add `events` as a dependency for these packages? I'm not sure how we generally solve this problem.

Reply:
If we did, nothing would use it. `browserify` would ignore it and polyfill it with its own `events` polyfill, and in a Node.js context it would use the built-in one.

We could add `events` here and force its use by importing it with a trailing slash (e.g. `import EventEmitter from 'events/'`), but I don't really see an advantage to that.

Elsewhere we've ignored this, and added to the README that the package is intended to be used in Node.js, or in an environment with an `events` polyfill such as that package.

Reply:
Okay, I've added ignores for the `events` import and updated READMEs for the packages that do this to note that it should either be available or polyfilled appropriately.